src/mvs_eland/C0_data_processing.py

```diff
@@ -1687,7 +1690,7 @@ def get_timeseries_multiple_flows(settings, dict_asset, file_name, header):
 
 def add_maximum_cap(dict_values, group, asset, subasset=None):
```

Comment:
Is there maybe a fraction of pytests for this function that you can write to make sure that what you intended here works?